docker/Dockerfile.cuda117.x86_64.deps

```diff
@@ -22,3 +22,13 @@ RUN NVJPEG2K_VERSION=0.5.0.25-1 && \
     cp /usr/include/nvjpeg2k* /usr/local/cuda/include/ && \
     cp /usr/lib/x86_64-linux-gnu/libnvjpeg2k* /usr/local/cuda/lib64/ && \
     rm -rf /var/lib/apt/lists/*
+
+RUN NVCOMP_VERSION=2.4 && \
```

Comment:
Can you merge it with L14 (where we install nvJPEG2k and libcufile)?

Reply:
done